README.md

```diff
@@ -1,45 +1,66 @@
+|                 | `bitrate`         | integer | optional  | Throughput of encoder in bits per second. Higher for better quality video, and lower for better storage efficiency. |
+|                 | `preset`          | string  | optional  | Name of codec video preset to use. See [here](https://trac.ffmpeg.org/wiki/Encode/H.264#a2.Chooseapresetandtune) for preset options.                                                                |
+| `cam_props`     |                   | object  | required  |                                                                                                   |
+|                 | `width`           | integer | required  | Width of the source camera frames in pixels.                                                      |
+|                 | `height`          | integer | required  | Height of the source camera frames in pixels.                                                     |
+|                 | `framerate`       | integer | required  | Number of frames per second provided by the source camera.                                        |
+
+### Example Configuration
 
 ```json
     {
-      "name": <video_store_component_name>,
+      "name": "video-store",
       "namespace": "rdk",
       "type": "camera",
       "model": "viam:video:storage",
       "attributes": {
-        "camera": <camera_component_name>, [required]
-        "sync": <data_manager_service_name>, [required]
-        "storage": { [required]
-            "segment_seconds": <length_of_video_segments>, [optional]
-            "size_gb": <total_storage_max_size>, [required]
-            "storage_path": <custom_path_to_store_video_files>, [optional]
-            "upload_path": <custom_path_to_upload_video_files>, [optional]
-        },
-        "video": { [optional]
-            "format": <video_format>, [optional]
-            "codec": <video_codec>, [optional]
-            "bitrate": <bits_pers_second>, [optional]
-            "preset": <video_preset>, [optional]
-        },
-        "cam_props": { [required]
-            "width": <pixel_width>, [required]
-            "height": <pixel_height>, [required]
-            "framerate": <frames_per_second>, [required]
+        "camera": "wc-cam"
+        "sync": "data-manager",
+        "storage": {
+            "segment_seconds": 10,
+            "size_gb": 50,
         },
+        "cam_props": {
+            "width": 640,
+            "height": 480,
+            "framerate": 25
+        }
       },
       "depends_on": [
-        <camera_component_name>, [required]
-        <data_manager_service_name> [required]
+        "wc-cam",
+        "data-manager"
       ]
     }
 ```
 
-Make sure to configure a [Data Manager Service](https://docs.viam.com/services/data/cloud-sync/) to uplaod video files to the cloud.
+### Configure a Data Manager Service
```

Comment:
Would suggest moving this up to above `configure a video-store` section in a section called ## Requirements and also adding the instructions to configure through the UI:

`Navigate to the [**CONFIGURE** tab](https://docs.viam.com/configure/) of your [machine](https://docs.viam.com/fleet/machines/) in [the Viam app](https://app.viam.com/).
[Add data-management to your machine](https://docs.viam.com/configure/#services).`

Reply:
Added requirements section.